scripts/sync_data.sh

```diff
@@ -11,4 +11,12 @@ PROD_OR_STAGING="${1:-prod}" # default prod
 S3_BUCKET="s3://${PROD_OR_STAGING}-elife-epp-data/data"
 DATA_FOLDER="$(realpath ${2:-${PARENT_DIR}/data})" # default ./data
 
+if [ "$PROD_OR_STAGING" == "prod" ]; then
+    ${SCRIPT_DIR}/preserve_pdfs.sh 2>&1
+
+    echo "Performing pre-sync checks before syncing with prod data s3"
+	${SCRIPT_DIR}/pre_sync_check.sh 2>&1
```

Comment:
I've noticed `pre_sync_check.sh` is also called in `preserve_psf.sh`, is this intentional?